engine/build.gradle

```diff
@@ -33,52 +33,58 @@ buildscript {
 }
 
 dependencies {
-    compile "com.badlogicgames.gdx:gdx:$gdxVersion"
-    compile "com.badlogicgames.gdx:gdx-box2d:$gdxVersion"
-    compile "com.badlogicgames.gdx:gdx-controllers:$gdxVersion"
+    api group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.25'
+    api group: 'com.google.code.gson', name: 'gson', version: '2.6.2'
```

Comment:
Where did these two come from?

Reply:
slf4j from desktop module. used in `engine`.
gson required for UIFormat and etc. idk where it was.